src/securityPolicy.h

```diff
@@ -18,6 +18,10 @@ security_policy_t policyForGetExtendedPublicKeyBulkExport(const bip44_path_t* pa
 security_policy_t policyForShowDeriveAddress(const addressParams_t* addressParams);
 security_policy_t policyForReturnDeriveAddress(const addressParams_t* addressParams);
 
+bool isNetworkUsual(uint32_t networkId, uint32_t protocolMagic);
+bool needsRunningScriptWarning(int32_t numCollaterals);
```

Comment:
I would prefer if functions in the header would be in the same order as they are in the .c file.

Reply:
That was my intent; it is hopefully fixed now.